src/certificates/certificates-card/certificate-signatories/signatory/Signatory.test.jsx

```diff
@@ -0,0 +1,73 @@
+import { render, waitFor, act } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { IntlProvider } from '@edx/frontend-platform/i18n';
+
+import { MODE_STATES } from '../../../data/constants';
+import messages from '../../messages';
+import Signatory from './Signatory';
+
+jest.mock('@edx/frontend-platform/i18n', () => ({
+  ...jest.requireActual('@edx/frontend-platform/i18n'),
+  useIntl: () => ({
+    formatMessage: (message) => message.defaultMessage,
+  }),
+}));
+jest.mock('@edx/frontend-platform/i18n', () => ({
+  ...jest.requireActual('@edx/frontend-platform/i18n'),
+  getConfig: () => ({ STUDIO_BASE_URL: 'http://localhost' }),
+}));
+
+const renderSignatory = (props) => render(
+  <IntlProvider locale="en">
+    <Signatory {...props} />
+  </IntlProvider>,
+);
+
+const defaultProps = {
+  id: 0,
+  name: 'John Doe',
+  title: 'Director',
+  organization: 'Organization',
+  signatureImagePath: '/path/to/image.png',
+  showDeleteButton: true,
+};
```

Comment:
[question]: Can we take the data from here [as a basis](src/certificates/__mocks__/signatories.js)?